dynawo/sources/Models/CPP/ModelNetwork/DYNModelTwoWindingsTransformer.cpp

```diff
@@ -363,18 +367,60 @@ ModelTwoWindingsTransformer::evalYMat() {
 }
 
 void
-ModelTwoWindingsTransformer::init(int& /*yNum*/) {
-  // not needed
+ModelTwoWindingsTransformer::init(int& yNum) {
+  if (!network_->isInitModel()) {
+    yOffset_ = (unsigned int) yNum;
```

Comment:
Please use the modern version static_cast<unsigned int> ( ... ). Moreover please add a sanity check assert(yNum >=0)

Reply:
done